hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/context/service-context.xml

```diff
@@ -80,6 +80,7 @@
         <property name="exceptionMappings">
             <map merge="true">
                 <entry key="org.alfresco.hxi_connector.hxi_extension.rest.api.exception.PredictionStateChangedException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_CONFLICT}" />
+                <entry key="org.alfresco.hxi_connector.common.exception.HxInsightConnectorRuntimeException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_INTERNAL_SERVER_ERROR}" />
```

Comment:
In the client you are throwing `WebScriptException` and here you have an declaration for `HxInsightConnectorRuntimeException`... this is correct? 🤔